### ISSUE TITLE
Anonymous access?

### ISSUE THREAD

**Issue by cool-nerd76** [Sonerezh/sonerezh#239]
Sorry, another follow up question. I had paid a third party developer change the app controller to allow anonymous access without having to login.. however, when that is enabled, then the player will not work as it seems to be wanting to create the playlist based on a non-existing username??.. if anybody can point me in the right direction on how to accomplish that.. that would be great. Thank you again.

**Comment by mandadimuralidharreddy** [comment]
Did you find out any possible way?

**Comment by cool-nerd76** [comment]
no, not at all.. i've since changed to another platform: Bemusic on CodeCanyon.. it's $39 but very much worth it.... I paid another $20 to have a script to import music files as it doesnt have an importer like Sonerezh does.. the script populates the database.
